loc/items/gem.js

```diff
@@ -2,76 +2,77 @@ export default [
   {
     name       : {en: 'Lucid' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
     }
   },
 
   {
     name       : {en: 'Melant' },
     description: {
-      en: 'Gem description'
+      en: 'Rough disc-shaped gray stone with strange druidical engravings encasing it.'
     }
   },
 
   {
     name       : {en: 'Turim' },
     description: {
-      en: 'Gem description'
+      en: ''
     }
   },
 
   {
     name       : {en: 'Fundo' },
     description: {
-      en: 'Gem description'
+      en: 'Brown desaturated gem, emitting putrid scents as you wear it around your neck.'
     }
   },
 
   {
     name       : {en: 'Royal' },
     description: {
-      en: 'Gem description'
+      en: 'Grand red gem enclosed in abyss dark fissures, granting your foes a sense of jealousy.'
     }
   },
 
   {
     name       : {en: 'Amary' },
     description: {
-      en: 'Gem description'
+      en: 'Lustrous blue artifact, exposed wispy white veins running along its spherical surface.'
     }
   },
 
   {
     name       : {en: 'Purum' },
     description: {
-      en: 'Gem description'
+      en: ''
     }
   },
 
   {
     name       : {en: 'Tara' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
     }
   },
 
   {
     name       : {en: 'Gloomy' },
     description: {
-      en: 'Gem description'
+      en: 'Small, smooth and dark, this gem is possesses treacherous traits as it slips from its wielders hands...'
     }
   },
 
   {
     name       : {en: 'Pluris' },
     description: {
-      en: 'Gem description'
+      en: 'Spectacular saturated pink plumes rise from this faceted prismical gem. Causes your foes to stare at you with awe.'
```

Comment:
Visual description is nice. I am not too sure about the "does XXX to your Foes" theme that goes through the descriptions, maybe try to insert it into the main sentence or try to make it more interesting, modify it a bit or add a sense of curiosity to it - not "does xyz" but "it is believed", etc.